osu.Game/Screens/OnlinePlay/Playlists/PlaylistsPlayer.cs

```diff
@@ -65,7 +65,7 @@ private void load()
             {
                 failed = true;
 
-                Logger.Error(e, "Failed to retrieve a score submission token.\n\nThis may happen if you are running an old or non-official release of osu! (ie. you are self-compiling).");
+                Logger.Log($"You are not able to submit a score: {e.Message}", LoggingTarget.Information, LogLevel.Important);
```

Comment:
This change results in a much worse error message in the case that a score submission token isn't issued by web (mentions a non-specific `WebException`). We'll be getting issue reports from people on old/self-compiled versions again.

Reply:
I've fallen back to the old display logic if the message is missing.